### PR TITLE
DOC-2045: Added TINY-9280 to the release notes.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2045

Changes:
* Added TINY-9280 to the release notes.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] ~`modules/ROOT/nav.adoc` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 1]
```diff
@@ -65,6 +65,14 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
...
 {productname} 6.5 also incorporates the following changes:
 
+=== `aria-activedescendant` attribute for `Typeahead` input component is updated to the menu item that is highlighted.
+//#TINY-9280
+
+In previous versions of {productname}, the Link Combobox screen reader feature lacked the capability to announce items within the dropdown. This meant that when users relied on screen readers to navigate through the dropdown list in the URL input combo box within the link dialog, there were no auditory notifications for the listed items.
+
+As a consequence, this significantly reduced the accessibility of the combo box for users that relying on screen readers.
+
+To address this issue, a fix was implemented to enable screen readers to announce the highlighted item within the dropdown. With this improvement, when users navigate through the dropdown, the screen reader will now audibly announce the value of the item currently highlighted.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.5-release-notes.adoc:75]:
```suggestion
=== Elements in the Insert → Link… dialog did not allow screen readers to announce listed items.

For example, the first UI element in the {productname} *Insert → Link…* dialog is a text-entry field labelled *URL*. When this element is navigated to using a screen reader, it is announced as a combobox and pressing the down arrow opens the combobox and presents available options.

In previous versions of {productname}, however, none of these available items were announced by the screen reader as they were selected, making these items effectively invisible to screen reader users.

With this update, the {productname} *Insert → Link…* dialog has been updated to properly support screen readers. When the URL combo box is opened, for example, all listed items are now announced by screen readers, as expected.
```

Suggested re-write.

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 2]
```diff
@@ -65,6 +65,13 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also incorporates the following changes:
 
+=== Elements in the Insert → Link… dialog did not allow screen readers to announce listed items.
+
```
Comment (kemister85) [inline modules/ROOT/pages/6.5-release-notes.adoc:69]:
```suggestion
//#TINY-9280

```

**Review by kemister85 — COMMENTED** [review]
inserting JIRA ref #

**Review by brian-forte — APPROVED** [review]

**Review by ArvinJ-H — APPROVED** [review]

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 3]
```diff
@@ -65,6 +65,14 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also incorporates the following changes:
 
+=== Elements in the Insert → Link… dialog did not allow screen readers to announce listed items.
+//#TINY-9280
+
+For example, the first UI element in the {productname} *Insert → Link…* dialog is a text-entry field labelled *URL*. When this element is navigated to using a screen reader, it is announced as a combobox and pressing the down arrow opens the combobox and presents available options.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/6.5-release-notes.adoc:71]:
```suggestion
For example, the URL text-entry field in the Link dialog is a combobox. When this element is focused with a screen reader, it is announced as a combobox and pressing the down arrow shows the available list items.
```

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 4]
```diff
@@ -65,6 +65,14 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
...
+
+In previous versions of {productname}, however, none of these available items were announced by the screen reader as they were selected, making these items effectively invisible to screen reader users.
+
+With this update, the {productname} *Insert → Link…* dialog has been updated to properly support screen readers. When the URL combo box is opened, for example, all listed items are now announced by screen readers, as expected.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/6.5-release-notes.adoc:75]:
```suggestion
With this update, all listed items in the combobox can be announced by the screen readers, making them more accessible.
```

**Review by tiny-ben-tran — APPROVED** [review]

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 5]
```diff
@@ -65,6 +65,14 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also incorporates the following changes:
 
+=== Elements in the Insert → Link… dialog did not allow screen readers to announce listed items.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.5-release-notes.adoc:68]:
```suggestion
=== List items in a combo box were not announced by screen readers.
```
Copy edit of @tiny-ben-tran’s suggested edit.

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 6]
```diff
@@ -65,6 +65,14 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
...
 {productname} 6.5 also incorporates the following changes:
 
+=== List items in a combo box were not announced by screen readers.
+//#TINY-9280
+
+For example, the first UI element in the {productname} *Insert → Link…* dialog is a text-entry field labelled *URL*. When this element is navigated to using a screen reader, it is announced as a combobox and pressing the down arrow opens the combobox and presents available options.
+
+In previous versions of {productname}, however, none of these items were announced by the screen reader as they were selected, making these items effectively invisible to screen reader users.
+
+With this update, the {productname} *Insert → Link…* dialog has been updated to properly support screen readers. When the URL combo box is opened, for example, all listed items are now announced by screen readers, as expected.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.5-release-notes.adoc:75]:
```suggestion
With this update, listed items in a combobox can be announced by screen readers, as expected.
```
Copy-edit of @tiny-ben-tran’s suggested edit.

**Review by brian-forte — APPROVED** [review]

**Pull request merged**